File: 백준/Bronze/34813. 공통교육과정/공통교육과정.java
import java.util.Scanner;
import java.util.HashMap;
import java.util.Map;
public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        String st = scan.nextLine();
        Map<Character,String> rlt=new HashMap<>();
        rlt.put('F',"Foundation");
        rlt.put('C',"Claves");
        rlt.put('V',"Veritas");
        rlt.put('E',"Exploration");
        System.out.print(rlt.get(st.charAt(0)));
    }
}

File: 백준/Bronze/34813. 공통교육과정/공통교육과정.c
#include <stdio.h>
struct cl {
    char ch;
    char word[12];
};
int main(void) {
    char st[9];
    scanf("%s",st);
    struct cl rlt[4]={{'F',"Foundation"},
    {'C',"Claves"},{'V',"Veritas"},{'E',"Exploration"}};
    for(int i=0;i<4;i++) {
        if(rlt[i].ch==st[0]) {
            printf("%s",rlt[i].word);
            break;
        }
    }
    return 0;
}